docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/LogisticRegression.cs

```diff
@@ -0,0 +1,100 @@
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of training data points.
+            var dataPoints = GenerateRandomDataPoints(1000);
+
+            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            var trainingData = mlContext.Data.LoadFromEnumerable(dataPoints);
+
+            // Define the trainer.
+            var pipeline = mlContext.BinaryClassification.Trainers.LbfgsLogisticRegression();
+
+            // Train the model.
+            var model = pipeline.Fit(trainingData);
+
+            // Create testing data. Use different random seed to make it different from training data.
+            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed:123));
+
+            // Run the model on test data set.
+            var transformedTestData = model.Transform(testData);
+
+            // Convert IDataView object to a list.
+            var predictions = mlContext.Data.CreateEnumerable<Prediction>(transformedTestData, reuseRowObject: false).ToList();
+
+            // Look at 5 predictions
+            foreach (var p in predictions.Take(5))
+                Console.WriteLine($"Label: {p.Label}, Prediction: {p.PredictedLabel}");
+
+            // Expected output:
+            //   Label: True, Prediction: True
+            //   Label: False, Prediction: True
+            //   Label: True, Prediction: True
+            //   Label: True, Prediction: True
+            //   Label: False, Prediction: True
+            
+            // Evaluate the overall metrics
+            var metrics = mlContext.BinaryClassification.Evaluate(transformedTestData);
+            SamplesUtils.ConsoleUtils.PrintMetrics(metrics);
+            
+            // Expected output:
+            //   Accuracy: 0.64
+            //   AUC: 0.70
+            //   F1 Score: 0.63
+            //   Negative Precision: 0.67
+            //   Negative Recall: 0.63
+            //   Positive Precision: 0.62
+            //   Positive Recall: 0.65
+            //   Log Loss: 0.90
+            //   Log Loss Reduction: 0.09
+            //   Entropy: 1.00
+        }
+
+        private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int seed=0)
+        {
+            var random = new Random(seed);
+            float randomFloat() => (float)random.NextDouble();
+            for (int i = 0; i < count; i++)
+            {
+                var label = randomFloat() > 0.5f;
+                yield return new DataPoint
+                {
+                    Label = label,
+                    // Create random features that are correlated with the label.
+                    // For data points with false label, the feature values are slightly increased by adding a constant.
+                    Features = Enumerable.Repeat(label, 50).Select(x => x ? randomFloat() : randomFloat() + 0.03f).ToArray()
```

Comment:
>0.03f [](start = 108, length = 5)

The accuracy is low. We want samples to ideally have 80% accuracy. Can we increase this to 0.1 and see how much accuracy improves. If so, let's make keep tree samples with old 0.03f value, and increase it for less powerful trainers.